src/client/actions/InviteCreate.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const Action = require('./Action');
+const Invite = require('../../structures/Invite');
+const { Events } = require('../../util/Constants');
+
+class InviteCreateAction extends Action {
+  handle(data) {
+    const client = this.client;
+    const inviteData = Object.assign(data, {
+      guild: client.guilds.get(data.guild_id),
+      channel: client.channels.get(data.channel_id),
+    });
+    const invite = new Invite(client, inviteData);
+    /**
+     * Emitted when an invite is created.
```

Comment:
This should have an `<info>` tag explaining that this event requires `MANAGE_GUILD` or `MANAGE_CHANNEL` (in the respective channel) to emit, to avoid confusion.

Reply:
my bad, totally missed that in the docs, thanks